cfgov/jinja2/v1/_includes/organisms/carousel.html

```diff
@@ -85,14 +85,14 @@ <h2>{{ item.title }}</h2>
 {% endmacro %}
 
 {% macro render( value ) %}
-<div class="o-carousel u-hidden">
-    <h5>{{ _( 'Featured' ) }}</h5>
+<div class="o-carousel u-hidden" aria-labelledby="o-carousel_heading">
+    <h2 class="h5" id="o-carousel_heading">{{ _( 'Featured content' ) }}</h2>
```

Comment:
We should check with UX whether this text should be changed. We could have `aria-label` on the `h2` alternatively.

Reply:
@sonnakim

Reply:
@anselmbradford works for me — either changing the H2, or just the aria label, to add the word "content"  👍

Reply:
We talked during team time about changing the aria label to "featured content"

Reply:
I think because this uses Django's translation framework (the usage of `_( )`), you'll need to change [the text translation file](https://github.com/cfpb/cfgov-refresh/blob/bc4c5941d49c1ab1c50f51af9d8a67815ac84e7f/cfgov/locale/es/LC_MESSAGES/django.po#L82-L83) to add a Spanish translation for this phrase, and then run [this step](https://cfpb.github.io/cfgov-refresh/translation/#4-run-the-compilemessages-management-command-to-compile-the-machine-object-file) to recompile the binary translation file.

Reply:
@chosak You raise a good point that we need to ensure we translate aria labels.